bolt/src/MCPlusBuilder.h

```diff
@@ -1348,6 +1365,13 @@ class MCPlusBuilder {
     return false;
   }
 
+  /// Store \p Target absolute adddress to \p RegName
+  virtual std::vector<MCInst> materializeAddress(const MCSymbol *Target, MCContext *Ctx,
+                                                 MCPhysReg RegName,
+                                                 int64_t Addend = 0) const {
+    llvm_unreachable("not implemented");
```

Comment:
This should still return something as some compilers may complain that the function does not return the declared return value, add a  "return {};" after llvm_unreachable. (I didn't get a warning from my compiler, though, so maybe compilers are getting smarter 😂 )

Reply:
Well, it kind of depends on how llvm_unreachable works. Actually I prefer builtin functions (__builtin_unreachable or __builtin_trap depending on the case), which are quite common today and compiler works fine with them :)